Status Code block retains error template colors even after successful command

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

Running a command that returns non 0 (like "false" for example) makes the prompt retain whatever template is set for on error, even after successful execution on subsequent commands.

example 1:
<img width="363" height="136" alt="Image" src="https://github.com/user-attachments/assets/5e81784e-a08e-48fc-a357-2e96cd66e9ab" />

example 2:
<img width="246" height="267" alt="Image" src="https://github.com/user-attachments/assets/8e96f6da-e6d5-4c30-969e-8d2d8bb0b9d6" />

Starting a new shell / `exec $SHELL` resets it.

### Theme

Example 1: default theme
Example 2: night-owl preset.

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
Version: 26.21.0

Shell: shell (5.2.37(1)-release)

Prompt:

╭─      0ms                                           shell  9.64%  15:05:41
╰─ 

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Root(false)                                -   0 ms
Path(true)                                 -   0 ms
Git(false)                                 -   0 ms
Executiontime(true)                        -   0 ms
Shell(true)                                -   0 ms
Sysinfo(true)                              -   1 ms
Angular(false)                             -   0 ms
Aurelia(false)                             -   0 ms
Aws(false)                                 -   0 ms
Az(false)                                  -   0 ms
Azfunc(false)                              -   0 ms
Cds(false)                                 -   0 ms
Crystal(false)                             -   0 ms
Cf(false)                                  -   0 ms
Cftarget(false)                            -   0 ms
Cmake(false)                               -   0 ms
Dart(false)                                -   0 ms
Dotnet(false)                              -   0 ms
Flutter(false)                             -   0 ms
Go(false)                                  -   0 ms
Haskell(false)                             -   0 ms
Java(false)                                -   0 ms
Julia(false)                               -   0 ms
Kotlin(false)                              -   0 ms
Kubectl(false)                             -   0 ms
Lua(false)                                 -   0 ms
Node(false)                                -   0 ms
Nx(false)                                  -   0 ms
Perl(false)                                -   0 ms
Php(false)                                 -   0 ms
Python(false)                              -   0 ms
R(false)                                   -   0 ms
Ruby(false)                                -   0 ms
Rust(false)                                -   0 ms
Swift(false)                               -   0 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms
Spotify(false)                             -   3 ms
Status(true)                               -   0 ms

Run duration: 8.391025ms

Cache path: /home/tudorc/.cache/oh-my-posh

Config path: no --config set, using default built-in configuration

Logs:

[DEBUG] 15:05:41.961 debug.go:39 → logging enabled, raw mode: true
[DEBUG] 15:05:41.961 store.go:]:135 → cache key not found: prompt_count_cache in store: session
[TRACE] 15:05:41.961 store.go:](session prompt_count_cache) - 20.539µs
[TRACE] 15:05:41.961 terminal.go:setPromptCount() - 25.518µs
[TRACE] 15:05:41.961 terminal.go:GOOS() - 60ns
[DEBUG] 15:05:41.961 terminal.go:setPwd:102 → /home/tudorc
[TRACE] 15:05:41.961 terminal.go:setPwd() - 13.926µs
[TRACE] 15:05:41.961 terminal.go:Init() - 46.658µs
[TRACE] 15:05:41.961 init.go:SessionID() - 310ns
[TRACE] 15:05:41.961 path.go:Path() - 5.7µs
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: template_cache in store: session
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: CONFIG_SOURCE in store: session
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: prompt_count_cache in store: session
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: CONFIG in store: session
[TRACE] 15:05:41.962 store.go:init(session bash.5ff02b12-73f6-49da-bd2e-9f168bd9b90d.omp.cache) - 366.404µs
[TRACE] 15:05:41.962 path.go:Path() - 81ns
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: INITVERSIONBASH in store: device
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: DSC_CONFIGURATION in store: device
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: is_wsl in store: device
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: upgrade_check in store: device
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: DSC_SHELL in store: device
[DEBUG] 15:05:41.962 store.go:init:92 → loading cache key: environment_platform in store: device
[TRACE] 15:05:41.962 store.go:init(device omp.cache) - 166.295µs
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: reload in store: session
[TRACE] 15:05:41.962 store.go:](session reload) - 4.629µs
[DEBUG] 15:05:41.962 store.go:]:146 → found cache key: CONFIG in store: session
[TRACE] 15:05:41.962 store.go:](session CONFIG) - 28.914µs
[TRACE] 15:05:41.962 gob.go:Get() - 32.632µs
[TRACE] 15:05:41.962 terminal.go:Shell() - 100ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 60ns
[DEBUG] 15:05:41.962 store.go:]:146 → found cache key: template_cache in store: session
[DEBUG] 15:05:41.962 store.go:]:153 → type mismatch for cache key: template_cache in store: session
[TRACE] 15:05:41.962 store.go:](session template_cache) - 6.993µs
[TRACE] 15:05:41.962 cache.go:restoreCache() - 13.054µs
[TRACE] 15:05:41.962 terminal_unix.go:Root() - 531ns
[TRACE] 15:05:41.962 terminal.go:Shell() - 30ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 31ns
[TRACE] 15:05:41.962 terminal.go:StatusCodes() - 60ns
[DEBUG] 15:05:41.962 store.go:]:146 → found cache key: is_wsl in store: device
[TRACE] 15:05:41.962 store.go:](device is_wsl) - 3.397µs
[TRACE] 15:05:41.962 terminal_unix.go:IsWsl() - 6.613µs
[TRACE] 15:05:41.962 terminal.go:Flags() - 31ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.962 store.go:]:146 → found cache key: is_wsl in store: device
[TRACE] 15:05:41.962 store.go:](device is_wsl) - 4.058µs
[TRACE] 15:05:41.962 terminal_unix.go:IsWsl() - 7.414µs
[TRACE] 15:05:41.962 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 40ns
[DEBUG] 15:05:41.962 terminal.go:User:236 → tudorc
[TRACE] 15:05:41.962 terminal.go:User() - 2.955µs
[DEBUG] 15:05:41.962 terminal.go:Host:253 → nobara-pc
[TRACE] 15:05:41.962 terminal.go:Host() - 4.338µs
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[DEBUG] 15:05:41.962 store.go:]:146 → found cache key: environment_platform in store: device
[TRACE] 15:05:41.962 store.go:](device environment_platform) - 9.188µs
[DEBUG] 15:05:41.962 terminal.go:Getenv:64 → 1
[TRACE] 15:05:41.962 terminal.go:Getenv(SHLVL) - 3.387µs
[DEBUG] 15:05:41.962 writer.go:Init:107 → terminal program: Unknown
[DEBUG] 15:05:41.962 writer.go:Init:108 → terminal shell: shell
[DEBUG] 15:05:41.962 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.962 terminal.go:Getenv(OMP_CACHE_DISABLED) - 2.795µs
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: accent_color in store: device
[TRACE] 15:05:41.962 store.go:](device accent_color) - 3.927µs
[TRACE] 15:05:41.962 colors.go:SetAccentColor() - 7.293µs
[TRACE] 15:05:41.962 terminal.go:Shell() - 80ns
[DEBUG] 15:05:41.962 terminal.go:Getenv:64 → 5.2.37(1)-release
[TRACE] 15:05:41.962 terminal.go:Getenv(POSH_SHELL_VERSION) - 2.975µs
[DEBUG] 15:05:41.962 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 15:05:41.962 text.go:patchTemplate:149 → {{ .Folder }}
[TRACE] 15:05:41.962 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.962 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.962 terminal.go:Getenv(POSH_CURSOR_LINE) - 2.475µs
[DEBUG] 15:05:41.962 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.962 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 7.594µs
[TRACE] 15:05:41.962 terminal.go:Flags() - 31ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 40ns
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 160ns
[DEBUG] 15:05:41.962 segment.go:Execute:108 → segment: Executiontime
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.962 store.go:](session toggle_cache) - 4.859µs
[DEBUG] 15:05:41.962 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[TRACE] 15:05:41.962 terminal.go:GOOS() - 60ns
[DEBUG] 15:05:41.962 segment.go:Execute:108 → segment: Root
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.962 store.go:](session toggle_cache) - 5.46µs
[DEBUG] 15:05:41.962 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.962 properties.go:GetBool:40 → always_enabled: false
[TRACE] 15:05:41.962 terminal.go:ExecutionTime() - 60ns
[DEBUG] 15:05:41.962 properties.go:GetFloat64:52 → threshold: 0.000000
[DEBUG] 15:05:41.962 properties.go:GetString:46 → style: roundrock
[TRACE] 15:05:41.962 terminal_unix.go:Root() - 421ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 471ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 40ns
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[TRACE] 15:05:41.962 terminal.go:GOOS() - 50ns
[DEBUG] 15:05:41.962 segment.go:Execute:108 → segment: Os
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.962 terminal.go:GOOS() - 80ns
[TRACE] 15:05:41.962 store.go:](session toggle_cache) - 4.208µs
[DEBUG] 15:05:41.962 segment.go:Execute:108 → segment: Path
[DEBUG] 15:05:41.962 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 store.go:](session toggle_cache) - 6.392µs
[DEBUG] 15:05:41.962 store.go:]:146 → found cache key: environment_platform in store: device
[DEBUG] 15:05:41.962 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.962 store.go:](device environment_platform) - 3.857µs
[DEBUG] 15:05:41.962 properties.go:GetBool:40 → display_distro_name: false
[DEBUG] 15:05:41.962 properties.go:GetBool:40 → display_cygpath: false
[TRACE] 15:05:41.962 terminal.go:GOOS() - 80ns
[DEBUG] 15:05:41.962 properties.go:GetString:46 → nobara: 
[TRACE] 15:05:41.962 separator.go:Separator() - 50ns
[TRACE] 15:05:41.962 terminal.go:Shell() - 151ns
[TRACE] 15:05:41.962 terminal.go:Shell() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[DEBUG] 15:05:41.962 properties.go:GetBool:40 → mapped_locations_enabled: true
[DEBUG] 15:05:41.962 properties.go:GetString:46 → windows_registry_icon: 
[DEBUG] 15:05:41.962 properties.go:GetString:46 → windows_registry_icon: 
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[TRACE] 15:05:41.962 separator.go:Separator() - 30ns
[DEBUG] 15:05:41.962 text.go:patchTemplate:149 →  {{ if .WSL }}WSL at {{ end }}{{.Data.Icon}}  
[TRACE] 15:05:41.962 terminal.go:GOOS() - 40ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 70ns
[DEBUG] 15:05:41.962 properties.go:GetString:46 → home_icon:   
[DEBUG] 15:05:41.962 properties.go:GetKeyValueMap:64 → mapped_locations: map[]
[TRACE] 15:05:41.962 separator.go:Separator() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 separator.go:Separator() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 180ns
[TRACE] 15:05:41.962 terminal.go:Flags() - 50ns
[DEBUG] 15:05:41.962 properties.go:GetStringArray:70 → cycle: []
[DEBUG] 15:05:41.962 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.962 properties.go:GetString:46 → folder_separator_template: 
[TRACE] 15:05:41.962 terminal.go:GOOS() - 60ns
[DEBUG] 15:05:41.962 properties.go:GetString:46 → folder_separator_icon: <#011627></> 
[DEBUG] 15:05:41.962 segment.go:Execute:108 → segment: Git
[DEBUG] 15:05:41.962 properties.go:GetBool:40 → cycle_folder_separator: false
[DEBUG] 15:05:41.962 properties.go:GetString:46 → folder_format: %s
[DEBUG] 15:05:41.962 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.962 properties.go:GetString:46 → edge_format: %s
[TRACE] 15:05:41.962 store.go:](session toggle_cache) - 5.771µs
[DEBUG] 15:05:41.962 properties.go:GetString:46 → left_format: %s
[DEBUG] 15:05:41.962 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.962 properties.go:GetString:46 → right_format: %s
[TRACE] 15:05:41.962 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.962 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.962 terminal.go:StackCount() - 50ns
[DEBUG] 15:05:41.962 terminal.go:ResolveSymlink:195 → /home/tudorc
[TRACE] 15:05:41.962 terminal.go:ResolveSymlink(/home/tudorc) - 11.732µs
[TRACE] 15:05:41.962 terminal_unix.go:DirIsWritable(/home/tudorc) - 4.689µs
[ERROR] 15:05:41.962 terminal.go:HasParentFilePath:464 → stat .git: no such file or directory
[TRACE] 15:05:41.962 terminal.go:HasParentFilePath(.git) - 37.29µs
[DEBUG] 15:05:41.962 text.go:patchTemplate:149 → {{ path .Data.Path .Data.Location }}
[DEBUG] 15:05:41.962 text.go:patchTemplate:149 →  {{ .Data.FormattedMs }}
[TRACE] 15:05:41.962 terminal.go:Shell() - 140ns
[TRACE] 15:05:41.963 terminal.go:Shell() - 30ns
[TRACE] 15:05:41.963 terminal.go:Flags() - 171ns
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[TRACE] 15:05:41.963 terminal.go:GOOS() - 90ns
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Aurelia
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 4.147µs
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.963 terminal.go:Flags() - 230ns
[TRACE] 15:05:41.963 terminal.go:Flags() - 91ns
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[ERROR] 15:05:41.963 terminal.go:FileContent:207 → open /home/tudorc/package.json: no such file or directory
[TRACE] 15:05:41.963 terminal.go:GOOS() - 90ns
[TRACE] 15:05:41.963 terminal.go:GOOS() - 101ns
[TRACE] 15:05:41.963 terminal.go:FileContent(package.json) - 9.909µs
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Shell
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Sysinfo
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.963 terminal.go:Flags() - 50ns
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 4.398µs
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 5.17µs
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.963 properties.go:GetKeyValueMap:64 → mapped_shell_names: map[]
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Az
[TRACE] 15:05:41.963 terminal.go:Shell() - 140ns
[TRACE] 15:05:41.963 terminal.go:Flags() - 40ns
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 7.784µs
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 properties.go:GetString:46 → source: cli|pwsh
[TRACE] 15:05:41.963 terminal.go:GOOS() - 41ns
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Azfunc
[TRACE] 15:05:41.963 terminal.go:Getenv(AZURE_CONFIG_DIR) - 3.837µs
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → extensions: [host.json local.settings.json function.json]
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → folders: []
[ERROR] 15:05:41.963 terminal.go:HasFilesInDir:123 → open .: no such file or directory
[DEBUG] 15:05:41.963 terminal.go:HasFilesInDir:124 → false
[DEBUG] 15:05:41.963 properties.go:GetBool:40 → home_enabled: false
[TRACE] 15:05:41.963 terminal.go:HasFilesInDir(azureProfile.json) - 23.795µs
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[ERROR] 15:05:41.963 terminal.go:HasFilesInDir:123 → open .: no such file or directory
[TRACE] 15:05:41.963 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 terminal.go:HasFilesInDir:124 → false
[TRACE] 15:05:41.963 terminal.go:GOOS() - 50ns
[TRACE] 15:05:41.963 terminal.go:HasFilesInDir(azureProfile.json) - 11.803µs
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Cds
[TRACE] 15:05:41.963 terminal.go:Getenv(POSH_AZURE_SUBSCRIPTION) - 3.527µs
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.963 terminal.go:Flags() - 40ns
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 4.428µs
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.963 text.go:patchTemplate:149 →   {{ .Data.Name }} 
[DEBUG] 15:05:41.963 properties.go:GetBool:40 → display_default: true
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 properties.go:GetString:46 → display_mode: context
[TRACE] 15:05:41.963 terminal.go:GOOS() - 90ns
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Crystal
[TRACE] 15:05:41.963 terminal.go:Getenv(AWS_VAULT) - 7.565µs
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → extensions: [.cdsrc.json .cdsrc-private.json *.cds]
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → folders: []
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 15.068µs
[DEBUG] 15:05:41.963 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[TRACE] 15:05:41.963 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → extensions: [*.cr shard.yml]
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.963 terminal.go:GOOS() - 30ns
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Cf
[DEBUG] 15:05:41.963 properties.go:GetBool:40 → home_enabled: false
[TRACE] 15:05:41.963 terminal.go:Getenv(AWS_DEFAULT_PROFILE) - 45.046µs
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.963 terminal.go:Flags() - 50ns
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 5.24µs
[TRACE] 15:05:41.963 terminal.go:Getenv(AWS_PROFILE) - 6.943µs
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.963 terminal.go:GOOS() - 50ns
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Cftarget
[TRACE] 15:05:41.963 terminal.go:Getenv(AWS_REGION) - 4.048µs
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → extensions: [manifest.yml mta.yaml]
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.963 terminal.go:Getenv(AWS_DEFAULT_REGION) - 3.597µs
[DEBUG] 15:05:41.963 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.963 terminal.go:Getenv:64 → NO DATA
[DEBUG] 15:05:41.963 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.963 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[TRACE] 15:05:41.963 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.963 terminal.go:Getenv(AWS_CONFIG_FILE) - 3.938µs
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[TRACE] 15:05:41.963 terminal.go:GOOS() - 81ns
[DEBUG] 15:05:41.963 segment.go:Execute:108 → segment: Cmake
[DEBUG] 15:05:41.963 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.963 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.963 store.go:](session toggle_cache) - 4.548µs
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.cmake CMakeLists.txt]
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 70ns
[ERROR] 15:05:41.964 terminal.go:FileContent:207 → open /home/tudorc/.aws/config: no such file or directory
[TRACE] 15:05:41.964 terminal.go:FileContent(/home/tudorc/.aws/config) - 519.814µs
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 40ns
[TRACE] 15:05:41.964 terminal.go:GOOS() - 70ns
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Dart
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[ERROR] 15:05:41.964 terminal.go:CommandPath:308 → exec: "cf": executable file not found in $PATH
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 terminal.go:CommandPath(cf) - 565.841µs
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 8.286µs
[TRACE] 15:05:41.964 terminal.go:GOOS() - 140ns
[TRACE] 15:05:41.964 terminal.go:HasCommand(cf) - 569.347µs
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Dotnet
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 terminal.go:Flags() - 40ns
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.dart pubspec.yaml pubspec.yml pubspec.lock]
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.871µs
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: [.dart_tool]
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.964 terminal.go:GOOS() - 100ns
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Flutter
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.cs *.csx *.vb *.sln *.slnf *.csproj *.vbproj *.fs *.fsx *.fsproj global.json]
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 terminal.go:Flags() - 40ns
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 4.398µs
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:GOOS() - 50ns
[TRACE] 15:05:41.964 terminal.go:Flags() - 40ns
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.dart pubspec.yaml pubspec.yml pubspec.lock]
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Go
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: [.dart_tool]
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 terminal.go:GOOS() - 50ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Haskell
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.951µs
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Java
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 11.322µs
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 3.898µs
[DEBUG] 15:05:41.964 properties.go:GetString:46 → stack_ghc_mode: never
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.hs *.lhs stack.yaml package.yaml *.cabal cabal.project]
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.964 terminal.go:Flags() - 50ns
[TRACE] 15:05:41.964 properties.go:GetBool:40 → home.336µs
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.964 terminal.go:GOOS() - 40ns
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [pom.xml build.gradle.kts build.sbt .java-version .deps.edn project.clj build.boot *.java *.class *.gradle *.jar *.clj *.cljc]
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Julia
[TRACE] 15:05:41.964 terminal.go:GOOS() - 30ns
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Kotlin
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 terminal.go:Flags() - 31ns
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.891µs
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 6.052µs
[TRACE] 15:05:41.964 terminal.go:GOOS() - 90ns
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Kubectl
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.jl]
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.kt *.kts *.ktm]
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → parse_kubeconfig: true
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 terminal.go:Getenv:64 → NO DATA
[TRACE] 15:05:41.964 terminal.go:Getenv(KUBECONFIG) - 2.575µs
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.964 terminal.go:Flags() - 60ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:GOOS() - 30ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Lua
[ERROR] 15:05:41.964 terminal.go:FileContent:207 → open /home/tudorc/.kube/config: no such file or directory
[TRACE] 15:05:41.964 terminal.go:GOOS() - 40ns
[TRACE] 15:05:41.964 terminal.go:FileContent(/home/tudorc/.kube/config) - 7.734µs
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Node
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.17µs
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 properties.go:GetString:46 → preferred_executable: lua
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.18µs
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.lua *.rockspec]
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: [lua]
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[TRACE] 15:05:41.964 terminal.go:GOOS() - 50ns
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Nx
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → display_error: false
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 3.477µs
[TRACE] 15:05:41.964 terminal.go:Flags() - 50ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.964 terminal.go:GOOS() - 70ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Perl
[TRACE] 15:05:41.964 terminal.go:GOOS() - 40ns
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [workspace.json nx.json]
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Php
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 4.038µs
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.19µs
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [.perl-version *.pl *.pm *.t]
[TRACE] 15:05:41.964 terminal.go:Flags() - 50ns
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.964 terminal.go:GOOS() - 40ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Python
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.php composer.json composer.lock .php-version blade.php]
[TRACE] 15:05:41.964 terminal.go:GOOS() - 30ns
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 4.148µs
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: R
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 properties.go:GetString:46 → display_mode: environment
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 3.988µs
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.R *.Rmd *.Rsx *.Rda *.Rd *.Rproj .Rproj.user]
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[TRACE] 15:05:41.964 terminal.go:GOOS() - 70ns
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Ruby
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 terminal.go:GOOS() - 30ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:GOOS() - 80ns
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.831µs
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Rust
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Swift
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 3.948µs
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 5.761µs
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.swift *.SWIFT Podfile]
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → extensions: [*.rs Cargo.toml Cargo.lock]
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 properties.go:GetStringArray:70 → folders: []
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:Flags() - 30ns
[DEBUG] 15:05:41.964 home.go:12 → /home/tudorc
[TRACE] 15:05:41.964 terminal.go:GOOS() - 80ns
[DEBUG] 15:05:41.964 segment.go:Execute:108 → segment: Time
[DEBUG] 15:05:41.964 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.964 store.go:](session toggle_cache) - 3.637µs
[DEBUG] 15:05:41.964 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.964 properties.go:GetString:46 → time_format: 15:04:05
[DEBUG] 15:05:41.964 text.go:patchTemplate:149 →  {{ round .Data.PhysicalPercentUsed .Data.Precision }}% 
[DEBUG] 15:05:41.964 text.go:patchTemplate:149 →  {{ .Data.CurrentDate | date .Data.Format }}
[DEBUG] 15:05:41.965 terminal_unix.go:TerminalWidth:83 → terminal width: 93
[TRACE] 15:05:41.965 terminal_unix.go:TerminalWidth() - 862.212µs
[TRACE] 15:05:41.965 terminal.go:Shell() - 80ns
[TRACE] 15:05:41.966 terminal.go:Shell() - 40ns
[TRACE] 15:05:41.966 terminal.go:Flags() - 170ns
[TRACE] 15:05:41.966 terminal.go:Flags() - 130ns
[DEBUG] 15:05:41.966 home.go:12 → /home/tudorc
[TRACE] 15:05:41.966 terminal.go:Flags() - 140ns
[DEBUG] 15:05:41.966 home.go:12 → /home/tudorc
[TRACE] 15:05:41.966 terminal.go:GOOS() - 161ns
[TRACE] 15:05:41.966 terminal.go:GOOS() - 60ns
[DEBUG] 15:05:41.966 segment.go:Execute:108 → segment: Text
[DEBUG] 15:05:41.966 segment.go:Execute:108 → segment: Spotify
[TRACE] 15:05:41.966 terminal.go:GOOS() - 60ns
[DEBUG] 15:05:41.966 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.966 segment.go:Execute:108 → segment: Status
[DEBUG] 15:05:41.966 store.go:]:135 → cache key not found: toggle_cache in store: session
[TRACE] 15:05:41.966 store.go:](session toggle_cache) - 10.109µs
[TRACE] 15:05:41.966 store.go:](session toggle_cache) - 10.079µs
[DEBUG] 15:05:41.966 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.966 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:05:41.966 store.go:]:135 → cache key not found: toggle_cache in store: session
[DEBUG] 15:05:41.966 store.go:]:146 → found cache key: is_wsl in store: device
[TRACE] 15:05:41.966 store.go:](session toggle_cache) - 11.181µs
[TRACE] 15:05:41.966 store.go:](device is_wsl) - 4.89µs
[DEBUG] 15:05:41.966 segment.go:isToggled:229 → no toggles found
[TRACE] 15:05:41.966 terminal_unix.go:IsWsl() - 11.783µs
[TRACE] 15:05:41.966 terminal.go:StatusCodes() - 70ns
[DEBUG] 15:05:41.966 properties.go:GetString:46 → status_template: {{ .Code }}
[DEBUG] 15:05:41.966 text.go:patchTemplate:149 → {{ .Code }}
[DEBUG] 15:05:41.966 properties.go:GetBool:40 → always_enabled: true
[DEBUG] 15:05:41.969 terminal.go:RunCommand:280 → Error org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
[TRACE] 15:05:41.969 terminal.go:RunCommand(bash -c dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get string:org.mpris.MediaPlayer2.Player string:PlaybackStatus | awk -F '"' '/string/ {print tolower($2)}') - 3.555742ms
[TRACE] 15:05:41.969 terminal.go:RunShellCommand() - 3.568586ms
[DEBUG] 15:05:41.969 text.go:patchTemplate:149 → {{ if .Data.Error }}#ef5350{{ end }}
[DEBUG] 15:05:41.969 text.go:patchTemplate:149 → {{ if .Data.Error }}#ef5350{{ end }}
[TRACE] 15:05:41.969 terminal.go:Shell() - 100ns
[TRACE] 15:05:41.969 terminal.go:Shell() - 30ns
[TRACE] 15:05:41.969 terminal.go:Flags() - 70ns
[TRACE] 15:05:41.969 terminal.go:Shell() - 30ns
[TRACE] 15:05:41.969 path.go:Path() - 60ns
[TRACE] 15:05:41.969 terminal.go:Flags() - 40ns
```

--- a/src/config/config.go
+++ b/src/config/config.go
@@ -50,39 +50,40 @@ const (
 
 // Config holds all the theme for rendering the prompt
 type Config struct {
-	Palette                 color.Palette          `json:"palette,omitempty" toml:"palette,omitempty" yaml:"palette,omitempty"`
-	DebugPrompt             *Segment               `json:"debug_prompt,omitempty" toml:"debug_prompt,omitempty" yaml:"debug_prompt,omitempty"`
-	Var                     map[string]any         `json:"var,omitempty" toml:"var,omitempty" yaml:"var,omitempty"`
-	Palettes                *color.Palettes        `json:"palettes,omitempty" toml:"palettes,omitempty" yaml:"palettes,omitempty"`
-	ValidLine               *Segment               `json:"valid_line,omitempty" toml:"valid_line,omitempty" yaml:"valid_line,omitempty"`
-	SecondaryPrompt         *Segment               `json:"secondary_prompt,omitempty" toml:"secondary_prompt,omitempty" yaml:"secondary_prompt,omitempty"`
-	TransientPrompt         *Segment               `json:"transient_prompt,omitempty" toml:"transient_prompt,omitempty" yaml:"transient_prompt,omitempty"`
-	ErrorLine               *Segment               `json:"error_line,omitempty" toml:"error_line,omitempty" yaml:"error_line,omitempty"`
-	Maps                    *maps.Config           `json:"maps,omitempty" toml:"maps,omitempty" yaml:"maps,omitempty"`
-	Upgrade                 *upgrade.Config        `json:"upgrade,omitempty" toml:"upgrade,omitempty" yaml:"upgrade,omitempty"`
-	Extends                 string                 `json:"extends,omitempty" toml:"extends,omitempty" yaml:"extends,omitempty"`
-	AccentColor             color.Ansi             `json:"accent_color,omitempty" toml:"accent_color,omitempty" yaml:"accent_color,omitempty"`
-	ConsoleTitleTemplate    string                 `json:"console_title_template,omitempty" toml:"console_title_template,omitempty" yaml:"console_title_template,omitempty"`
-	PWD                     string                 `json:"pwd,omitempty" toml:"pwd,omitempty" yaml:"pwd,omitempty"`
-	Source                  string                 `json:"-" toml:"-" yaml:"-"`
-	Format                  string                 `json:"-" toml:"-" yaml:"-"`
-	TerminalBackground      color.Ansi             `json:"terminal_background,omitempty" toml:"terminal_background,omitempty" yaml:"terminal_background,omitempty"`
-	ToolTipsAction          Action                 `json:"tooltips_action,omitempty" toml:"tooltips_action,omitempty" yaml:"tooltips_action,omitempty"`
-	Tooltips                []*Segment             `json:"tooltips,omitempty" toml:"tooltips,omitempty" yaml:"tooltips,omitempty"`
-	Blocks                  []*Block               `json:"blocks,omitempty" toml:"blocks,omitempty" yaml:"blocks,omitempty"`
-	ITermFeatures           terminal.ITermFeatures `json:"iterm_features,omitempty" toml:"iterm_features,omitempty" yaml:"iterm_features,omitempty"`
+	Palette                 color.Palette   `json:"palette,omitempty" toml:"palette,omitempty" yaml:"palette,omitempty"`
+	DebugPrompt             *Segment        `json:"debug_prompt,omitempty" toml:"debug_prompt,omitempty" yaml:"debug_prompt,omitempty"`
+	Var                     map[string]any  `json:"var,omitempty" toml:"var,omitempty" yaml:"var,omitempty"`
+	Palettes                *color.Palettes `json:"palettes,omitempty" toml:"palettes,omitempty" yaml:"palettes,omitempty"`
+	ValidLine               *Segment        `json:"valid_line,omitempty" toml:"valid_line,omitempty" yaml:"valid_line,omitempty"`
+	SecondaryPrompt         *Segment        `json:"secondary_prompt,omitempty" toml:"secondary_prompt,omitempty" yaml:"secondary_prompt,omitempty"`
+	TransientPrompt         *Segment        `json:"transient_prompt,omitempty" toml:"transient_prompt,omitempty" yaml:"transient_prompt,omitempty"`
+	ErrorLine               *Segment        `json:"error_line,omitempty" toml:"error_line,omitempty" yaml:"error_line,omitempty"`
+	Maps                    *maps.Config    `json:"maps,omitempty" toml:"maps,omitempty" yaml:"maps,omitempty"`
+	Upgrade                 *upgrade.Config `json:"upgrade,omitempty" toml:"upgrade,omitempty" yaml:"upgrade,omitempty"`
+	TerminalBackground      color.Ansi      `json:"terminal_background,omitempty" toml:"terminal_background,omitempty" yaml:"terminal_background,omitempty"`
+	AccentColor             color.Ansi      `json:"accent_color,omitempty" toml:"accent_color,omitempty" yaml:"accent_color,omitempty"`
+	ConsoleTitleTemplate    string          `json:"console_title_template,omitempty" toml:"console_title_template,omitempty" yaml:"console_title_template,omitempty"`
+	PWD                     string          `json:"pwd,omitempty" toml:"pwd,omitempty" yaml:"pwd,omitempty"`
+	Source                  string          `json:"-" toml:"-" yaml:"-"`
+	Format                  string          `json:"-" toml:"-" yaml:"-"`
+	Extends                 string          `json:"extends,omitempty" toml:"extends,omitempty" yaml:"extends,omitempty"`
+	ToolTipsAction          Action          `json:"tooltips_action,omitempty" toml:"tooltips_action,omitempty" yaml:"tooltips_action,omitempty"`
+	base64                  string
 	Cycle                   color.Cycle            `json:"cycle,omitempty" toml:"cycle,omitempty" yaml:"cycle,omitempty"`
+	ITermFeatures           terminal.ITermFeatures `json:"iterm_features,omitempty" toml:"iterm_features,omitempty" yaml:"iterm_features,omitempty"`
+	Blocks                  []*Block               `json:"blocks,omitempty" toml:"blocks,omitempty" yaml:"blocks,omitempty"`
+	Tooltips                []*Segment             `json:"tooltips,omitempty" toml:"tooltips,omitempty" yaml:"tooltips,omitempty"`
 	Version                 int                    `json:"version" toml:"version" yaml:"version"`
-	EnableCursorPositioning bool                   `json:"enable_cursor_positioning,omitempty" toml:"enable_cursor_positioning,omitempty" yaml:"enable_cursor_positioning,omitempty"`
-	ShellIntegration        bool                   `json:"shell_integration,omitempty" toml:"shell_integration,omitempty" yaml:"shell_integration,omitempty"`
-	MigrateGlyphs           bool                   `json:"-" toml:"-" yaml:"-"`
 	Async                   bool                   `json:"async,omitempty" toml:"async,omitempty" yaml:"async,omitempty"`
+	MigrateGlyphs           bool                   `json:"-" toml:"-" yaml:"-"`
+	ShellIntegration        bool                   `json:"shell_integration,omitempty" toml:"shell_integration,omitempty" yaml:"shell_integration,omitempty"`
 	FinalSpace              bool                   `json:"final_space,omitempty" toml:"final_space,omitempty" yaml:"final_space,omitempty"`
 	UpgradeNotice           bool                   `json:"-" toml:"-" yaml:"-"`
 	updated                 bool
 	extended                bool
 	PatchPwshBleed          bool `json:"patch_pwsh_bleed,omitempty" toml:"patch_pwsh_bleed,omitempty" yaml:"patch_pwsh_bleed,omitempty"`
 	AutoUpgrade             bool `json:"-" toml:"-" yaml:"-"`
+	EnableCursorPositioning bool `json:"enable_cursor_positioning,omitempty" toml:"enable_cursor_positioning,omitempty" yaml:"enable_cursor_positioning,omitempty"`
 }
 
 func (cfg *Config) MakeColors(env runtime.Environment) color.String {

--- a/src/config/gob.go
+++ b/src/config/gob.go
@@ -19,7 +19,7 @@ func (cfg *Config) Store() {
 	defer log.Trace(time.Now())
 
 	cache.Set(cache.Session, configSourceKey, cfg.Source, cache.INFINITE)
-	cache.Set(cache.Session, configKey, cfg, cache.INFINITE)
+	cache.Set(cache.Session, configKey, cfg.Base64(), cache.INFINITE)
 }
 
 func Get(configFile string, reload bool) *Config {
@@ -32,17 +32,27 @@ func Get(configFile string, reload bool) *Config {
 		}
 	}
 
-	cfg, found := cache.Get[*Config](cache.Session, configKey)
+	base64String, found := cache.Get[string](cache.Session, configKey)
 	if !found {
 		log.Debug("no cached config found")
 		return Load(configFile, false)
 	}
 
-	return cfg
+	var cfg Config
+	if err := cfg.Restore(base64String); err != nil {
+		log.Debug("failed to restore config from cache")
+		return Load(configFile, false)
+	}
+
+	return &cfg
 }
 
 func (cfg *Config) Base64() string {
 	defer log.Trace(time.Now())
+
+	if cfg.base64 != "" {
+		return cfg.base64
+	}
 
 	var buffer bytes.Buffer
 	encoder := gob.NewEncoder(&buffer)
@@ -52,5 +62,29 @@ func (cfg *Config) Base64() string {
 		return ""
 	}
 
-	return base64.StdEncoding.EncodeToString(buffer.Bytes())
+	cfg.base64 = base64.StdEncoding.EncodeToString(buffer.Bytes())
+	return cfg.base64
+}
+
+func (cfg *Config) Restore(base64String string) error {
+	defer log.Trace(time.Now())
+
+	data, err := base64.StdEncoding.DecodeString(base64String)
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+
+	var buffer bytes.Buffer
+	buffer.Write(data)
+	decoder := gob.NewDecoder(&buffer)
+	err = decoder.Decode(cfg)
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+
+	cfg.base64 = base64String
+
+	return nil
 }

